docs/usage/configuration/config_documentation.md

```diff
@@ -3928,6 +3928,11 @@ This setting has the following sub-options:
    localhost and 6379
 * `password`: Optional password if configured on the Redis instance.
 * `dbid`: Optional redis dbid if needs to connect to specific redis logical db.
+* `use_ssl`: Whether to use ssl connection. Defaults to false.
```

Comment:
This should be `use_tls` — TLS is the 'new' name for what used to be called SSL.

However I think you're right to have a separate toggle for it, since all the below options are optional.

Reply:
Oh indeed good point, I don't know how I missed that.